docs/en/concept/connector-v2-features.md

```diff
@@ -69,3 +69,10 @@ For sink connector, the sink connector supports exactly-once if any piece of dat
 ### cdc(change data capture)
 
 If a sink connector supports writing row kinds(INSERT/UPDATE_BEFORE/UPDATE_AFTER/DELETE) based on primary key, we think it supports cdc(change data capture).
+
+### support multiple table write
+
+Supports write multiple tables in one SeaTunnel job, users can dynamically specify the table's identifier by [configuring placeholders](./sink-options-placeholders.md).
+
+Requires:
+- Make sure the sink connector you are using has implemented `SupportMultiTableSink`、`SupportMultiTableSinkWriter`、`SupportMultiTableSinkAggregatedCommitter`
```

Comment:
It is strange to let users judge whether the corresponding interface is implemented here, because users are not developers. We can maintain the list of all sinks that support multiple tables through code. When a new connector that supports multiple tables is detected, it is mandatory to update the corresponding document, otherwise CI will report an error.

Reply:
Agree with you